src/nidmm/nidmm.mak

```diff
@@ -6,7 +6,7 @@ MODULE_FILES_TO_GENERATE := $(DEFAULT_PY_FILES_TO_GENERATE)
 
 MODULE_FILES_TO_COPY := $(DEFAULT_PY_FILES_TO_COPY)
 
-RST_FILES_TO_GENERATE := $(DEFAULT_RST_FILES_TO_GENERATE)
+RST_FILES_TO_GENERATE := $(filter-out rep_caps.rst,$(DEFAULT_RST_FILES_TO_GENERATE))
```

Comment:
It's a shame that this knowledge (the fact that this driver doesn't have repeated capabilities) is duplicated - since it is also present in config.py.

That said, I think it's better to use additive rather than substractive logic:

* DEFAULT_RST_FILES_TO_GENERATE could be renamed to RST_FILES_FOR_ALL_MODULES
* On those that *do* have repeated capability, add rep_caps.rst

This may mean modifying more .mak since most drivers have repeated capability. I think the clarity is worth it. This has a WTH factor to it.

Reply:
We have been filtering out from the beginning so this is not new. If we make the change only for rep_caps.rst then it doesn't match the others, and if we change all of them, there will be unrelated changes in this PR. If you want to make this change, it should be a separate PR after this one is merged (or just closed).

Current filter out stats:
`rep_caps.rst`: nidmm, nimodinst, nise, nitclk
`enums.py`: nimodinst, nitclk
`_attributes.py`: nimodinst, nise
`enums.rst`: nimodinst, nitclk

Changing to additive rather than subtractive would mean adding these files to all the others not listed for that particular file.